src/chatgpt-operation/api.js

```diff
@@ -1,50 +1,54 @@
-import { Configuration, OpenAIApi } from "openai";
-
+const OpenAI = require("openai");
```

Comment:
Can we not mix CJS and ESM? Can't imagine a newer version of `openai` suddenly relies on cjs 🤔

Reply:
Hey!

For some reason if I used this:

`import OpenAI from 'openai';`

I would run into this while building the extension:

`Building Directus extension...node_modules/openai/core.mjs (1:30) The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten`

Reply:
unfortunately we'll have to find some way of fixing this, using `require` means that the `openai` dependency is not bundled with the extension meaning it will error if `openai` is not installed in the directus dependency tree
![image](https://github.com/user-attachments/assets/6687a2cb-0729-4b26-a81c-26a246a59b8e)